packages/codemods/src/codeMods/__tests__/utilities/importUtils.test.ts

```diff
@@ -0,0 +1,72 @@
+import { getImportsByPath, repathImport } from '../../utilities';
+import { Project } from 'ts-morph';
+
+const fileName = 'mockImports.tsx';
+const rootPath = 'office-ui-fabric-react';
+const buttonPath = `${rootPath}/lib/Button`;
+
+describe('Import Utilities test', () => {
+  let project: Project;
+
+  beforeEach(() => {
+    project = new Project();
+    project.addSourceFilesAtPaths(`${process.cwd()}/**/__tests__/mock/utils/*.tsx`);
```

Comment:
we should remove these files after test?

Reply:
"Add source files" doesn't mean actually creating files, just adding references to pre-existing mock files.